src/main/java/mate.academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Ensure that the Injector class is correctly implemented and configured to return an instance of ProductService. If the Injector class is not properly set up, this line may throw a runtime exception.